elliott

```diff
@@ -158,7 +167,7 @@ Bugzilla Bugs.
               help="Create the advisory (by default only a preview is displayed)")
 @pass_runtime
 @click.pass_context
-def create(ctx, runtime, kind, impetus, date, assigned_to, manager, package_owner, yes):
+def create(ctx, runtime, errata_type, kind, impetus, date, assigned_to, manager, package_owner, impact, cve, yes):
```

Comment:
If `impact` and `cve` are auto detected by the CVE bug sweep, they might not be required to be passed to the create function.